tests/slug_ref.bats

```diff
@@ -42,6 +42,12 @@
     "an-awesome-feature-very-very-very-very-very-very-very-long-more"
 }
 
+@test "Short long hash" {
```

Comment:
Can you move this test and the `test_short_sha` function to a dedicated `bats` file: `tests/short_sha.bats`?